src/CoreBundle/Service/MaintenanceModeServiceInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Service;
+
+use ChameleonSystem\CoreBundle\Exception\MaintenanceModeErrorException;
+
+interface MaintenanceModeServiceInterface
+{
+    /**
+     * Returns if the maintenance mode is activated (by file).
```

Comment:
Client code should not need to care how maintenance mode works internally, so I'd remove the reference to the file.

Reply:
Removed from interface for now: will be changed for #119